src/views/AnnualReport.vue

```diff
@@ -170,7 +178,7 @@
                   <h2 id="directors-header">
                     3. Directors
                   </h2>
-                  <p v-if="allowChange('cod')">
+                  <p v-if="allowChange('cod') && IsAuthorized(AuthorizedActions.DIRECTOR_CHANGE_FILING)">
```

Comment:
Whoops, I just noticed this.

Please remove this check (and also the check on lines 139 and 140). This authorized action is for Director Change filings on their own. When director changes or address changes are part of an AR, they're authorized (if the AR is authorized).

Reply:
Checks are removed. Resolving.